data_prep/scotland/get_reference_layers.sh

```diff
@@ -80,6 +80,39 @@ function population {
   tippecanoe tmp/population.geojson -zg --generate-ids -l population -o $OUT/population.pmtiles
 }
 
+function stats19 {
+  # Use code from https://github.com/acteng/atip-data-prep/tree/main/stats19
+  git submodule init
+  git submodule update
+
+  cd atip-data-prep/stats19
+  mkdir -p input tmp
+
+  # URLs from https://www.data.gov.uk/dataset/cb7ae6f0-4be6-4935-9277-47e5ce24a11f/road-safety-data
+  if [ ! -e "input/dft-road-casualty-statistics-casualty-1979-latest-published-year.csv" ]; then
```

Comment:
No biggie, but you can probably skip the -e check.

In case you missed it, `download_to_subdir` uses `--timstamping` option, so it won't actually download the file contents unless the server reports it as modified vs. your local versions timestamps.

Reply:
Oh nice! Done